### ISSUE TITLE
`npx remotion lambda render` fails to wait for output to be created

### ISSUE THREAD

**Issue by taylorhughes** [remotion-dev/remotion#2318]
# Bug Report 🐛

Attempted to render a video on lambda:
```
taylorhughes$  npx remotion lambda sites create ideatorjs/render/renderIndex.ts --site-name=highlight-test-3
...

taylorhughes$  npx remotion lambda render https://remotionlambda-useast1-ub9jaght1s.s3.us-east-1.amazonaws.com/sites/highlight-test-3/index.html ClipComposition out-name.mp4
bucket = remotionlambda-useast1-ub9jaght1s, function = remotion-render-3-3-95-mem2048mb-disk2048mb-120sec
renderId = v25r7r27im, codec = h264 (derived from download name)
(1/6)                       Invoked lambdas 0% 
(2/6)                       Rendering frames
(3/6)                       Encoding chunks 0%
taylorhughes$ 
taylorhughes$ echo $?
1
```

Quits unceremoniously with no error output after step 3/6. Same happens without an out-name.mp4 argument (quits at step 3/5).

By adding a bunch of prints into the command source it appears that `getRenderProgress` (`api/get-render-progress.js`) fails silently with an error: 
```
lambda Function.ResponseSizeTooLarge: Response payload size exceeded maximum allowed payload size
```

But the video still gets rendered successfully in the S3 bucket under `<render_id>/out.mp4`. No idea what's going on beyond that or how to debug further.

Versions:
MacBook M1, MacOS 13.3.1
node v16.14.2
@remotion/lambda 3.3.95

**Comment by JonnyBurger** [comment]
Thanks for the report!
This seems critical, will deal with it tomorrow with the highest prio.

**Comment by taylorhughes** [comment]
Confirmed the bug is resolved in 3.3.96 thank you so much! 🙏

**Comment by JonnyBurger** [comment]
Excellent, thanks for confirming! 🙌
